src/app/src/main/kotlin/org/vaccineimpact/api/app/repositories/jooq/JooqTouchstoneRepository.kt

```diff
@@ -69,8 +68,8 @@ class JooqTouchstoneRepository(
 
         val referenceRecord = records.firstOrNull()
 
-        val source = referenceRecord?.get(DEMOGRAPHIC_SOURCE.NAME)
-        val gender =  referenceRecord?.get(GENDER.NAME)
+        val source = referenceRecord?.get(field(name("s", "sourceCode"), String::class.java))
+        val gender = referenceRecord?.get(GENDER.NAME)
```

Comment:
Would it be worth writing a generic inline function to make this code a little more readable?

e.g.

```
inline fun <reified T: Any?> Record.get(field: Field<*>): T = this.get(field, T::class.java)
```

Then you can replace the `val source = ...` line with:

```
referenceRecord?.get<String>(field(name("s", "sourceCode")))
```

Which at least removes the need for the `::class.java`